api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -294,7 +301,7 @@ public static boolean matchRequiredVersions(String version, String versionRange)
 			for (String range : ranges) {
 				// need to externalize this string
 				String separator = "-";
-				if (range.indexOf("*") > 0 || range.indexOf(separator) > 0 && (!isVersionWithQualifier(range))) {
+				if (range.indexOf("+") > 0 || range.indexOf("*") > 0 || range.indexOf(separator) > 0 && (!isVersionWithQualifier(range))) {
```

Comment:
Hello @jnsereko since I only use each instance of the index only one, I think that encapsulating them in variables would only add unnecessary complexity and it would have any notable benefits.